lib/utils/telemetry/generatePayload.js

```diff
@@ -119,12 +119,21 @@ module.exports = async (serverless = null) => {
     return resolveIsLocallyInstalled();
   })();
 
-  const { command } = resolveInput();
+  const { command, options, commandSchema } = resolveInput();
+
+  // We only consider options that are present in schema, with the exclusion of
+  // `region` and `stage`, because they are always present due to mutation of `options` object
```

Comment:
I there's a mutation, let's fix it. I think we can easily fix it by ensure that we store copy of `options` in `Serverless` constructor.

Reply:
That's a good call, fixed